updater/reports/ReportFailedWebhooks.py

```diff
@@ -0,0 +1,14 @@
+from .ReportDaily import *
+
+# Report how many webhooks failed
+class ReportFailedWebhooks(ReportDaily):
+	def name(self):
+		return "failed-webhooks"
+
+	def updateDailyData(self):
+		self.detailedHeader, self.detailedData = self.parseData(
```

Comment:
```suggestion
		self.detailedHeader, newData = self.parseData(
```
This is necessary for the change below